import "mapreduce/yt/interface/protos/extension.proto";

message TReducerOutputRow {
    optional uint64 Index = 1 [(NYT.column_name) = "idx"];
    optional uint64 Count = 2 [(NYT.column_name) = "cnt"];
    optional string Subnet = 3 [(NYT.column_name) = "subnet"];
}
